### PR TITLE
Set PROJ_NETWORK=OFF if proj-data included

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/scripts/activate.sh [hunk 1]
```diff
@@ -8,10 +8,15 @@ if [ -n "$PROJ_LIB" ]; then
...
+  export PROJ_LIB="${CONDA_PREFIX}/Library/share/proj"
 fi
 
-export PROJ_NETWORK="ON"
+if [ -f "${CONDA_PREFIX}/share/proj/copyright_and_licenses.csv" ]; then
+  # proj-data is installed because its license was copied over
+  export PROJ_NETWORK="OFF"
+else
+  export PROJ_NETWORK="ON"
+fi
```
Comment (xylar) [inline recipe/scripts/activate.sh:22]: @ocefpaf and @snowman2, I realize this isn't the most elegant way to figure out if `proj-data` is installed.  We could create some other file such as `.proj-data` that gets put into this folder to indicate that `proj-data` has been installed.  What do you think.

**Comment by xylar** [comment]
This is to address https://github.com/conda-forge/proj-data-feedstock/issues/4

**Comment by xylar** [comment]
It worked as expected in my local test under linux with bash:
```
$ conda create -y -n test --use-local proj proj-data
$ conda activate test
$ env | grep PROJ
PROJ_LIB=/home/xylar/miniconda3/envs/test/share/proj
PROJ_NETWORK=OFF
```

Update: I wanted to check that the env without `proj-data` also behaves as it should:
```
$ conda create -y -n test proj
$ conda activate test
$ env | grep PROJ
PROJ_LIB=/home/xylar/miniconda3/envs/test/share/proj
PROJ_NETWORK=ON
```
Looks good!

File: recipe/scripts/activate.sh (hunk 1 above)
Comment (ocefpaf) [reply recipe/scripts/activate.sh:22]: Sounds good. But I'm also OK with what you did here. It is not elegant but "good enough."

Comment (xylar) [reply recipe/scripts/activate.sh:22]: If it's good enough, I think I'll probably stick with this.

**Pull request merged**